proposals/stage-1/PROJECT_ONBOARDING/README.md

```diff
@@ -3,30 +3,51 @@
 
 ## Champion
 
-Jory Burson (@jorydotcom) & Michael Dawson (@mhdawson)
+Jory Burson (@jorydotcom), Michael Dawson (@mhdawson), & Myles Borins (@MylesBorins)
 
 ## Description
 
-Proposal creates a checklist artifact to be used to onboard projects that have been accepted into the OpenJS Foundation.
+This proposal codifies the process for new projects applying to join the foundation. It includes a project application template and a checklist to be used to onboard projects as part of being accepted into the OpenJS Foundation.
 
 ## Required Resources
 
-Discussion/approval by bootstrap committee
+* General support from the CPC
+  - Keep track of applying projects
+  - Ongoing communication with applying projects
+  - Mentoring projects as they onboard
+* General support from the CPC Director
+  - Updates to Board of Directors of OpenJS Foundation
+* Project Management support from OpenJS Foundation.
+  - Keep track of applying projects
+  - Ongoing communication with project
 
 ## Who would be responsible?
 
-This document would belong to the CPC to update and use moving forward.
+This document would belong to the CPC to update and use moving forward. Large changes may require board approval.
 
 ## How would success be measured?
 
 Initially we should measure this document's success by how well it helps us onboard our current projects into the new Foundation system. We expect that it might also be successfully used to help would-be foundation projects understand what to expect after joining.
 
 ## Why this proposal is important
 
-We need to share a workflow and understanding of what steps are required to bring in projects to the OpenJS Foundation. This checklist is an attempt to organize our ideas and our work to make that happen effectively.
+We need to share a workflow and understanding of what steps are required to bring in projects to the OpenJS Foundation. This process is an attempt to organize our ideas and our work to make that happen effectively. This process should be explicit and consistent to make applications fair for all projects.
 
-## What is neccessary to complete this proposal
+## Unresolved Question
 
-Agreement on the initial checklist items; interim owners/helpers for each of the tasks.
+* Should there be a quiet period?
+* Should all emeritus projects be expected to go through this process?
+* Should projects be able to enter as `Impact Level`?
+
+## What is necessary to complete this proposal
 
+Agreement on the initial checklist items; interim owners/helpers for each of the tasks.
 
+- [x] Template for Project Applications
+- [x] Onboarding Check List
+- [ ] Update to Project Progression
+- [ ] Potential changes to CPC Charter
+- [ ] Discussion/approval by bootstrap committee
+- [ ] Discussion/approval by the Board
+- [ ] Adoption by the CPC
+- [ ] Run existing projects through the process
```

Comment:
What would be the desired outcome of  “run existing projects through the project”?

Reply:
Can you moved this question to https://github.com/openjs-foundation/bootstrap/pull/158?